docs/cog_guides/admin.rst

```diff
@@ -0,0 +1,380 @@
+    Make sure it doesn't give extra perms or anything that can break
+    your server's security.
+
+**Arguments**
+
+* ``<role>``: The role to add to the list. |role-input|
+
+.. _admin-command-selfrole-delete:
+
+"""""""""""""""
+selfrole delete
+"""""""""""""""
+
+.. note:: This command is locked to the member with the ``Manage roles``
+    permission.
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]selfrole add <role>
+
+**Description**
+
+Removes a role from the list of selfroles.
+
+**Arguments**
+
+* ``<role>``: The role to remove from the list. |role-input|
+
+.. _admin-command-addrole:
+
+^^^^^^^
+addrole
+^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]addrole <rolename> [user]
+
+**Description**
+
+Adds a role to a member. If ``user`` is not given, it will be considered
+as yourself, the command author.
+
+**Arguments**
+
+* ``<role>``: The role to add to the member. |role-input|
+
+* ``[user=ctx]``: The member you want to add the role to. Defaults to the
+  command author. |member-input|
+
+.. _admin-command-removerole:
+
+^^^^^^^^^^
+removerole
+^^^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]removerole
+
+**Description**
+
+Removes a role from a member. If ``user`` is not given, it will be considered
+as yourself, the command author.
+
+**Arguments**
+
+* ``<role>``: The role to remove. |role-input|
+
+* ``[user=ctx]``: The member to remove the role from. |member-input| Defaults to
+  the command author.
+
+.. _admin-command-editrole:
+
+^^^^^^^^
+editrole
+^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole
+
+**Description**
+
+Edits settings of a role.
+
+.. _admin-command-editrole-name:
+
+"""""""""""""
+editrole name
+"""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole name <role> <name>
+
+**Description**
+
+Edits the name of a role.
+
+**Arguments**
+
+* ``<role>``: The role name to edit. |role-input|
+
+* ``<name>``: The new role name.
+
+.. _admin-command-editrole-color:
+
+""""""""""""""
+editrole color
+""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole color <role> <color>
+
+**Description**
+
+Edits the color of a role.
+
+**Arguments**
+
+* ``<role>``: The role name to edit. |role-input|
+
+* ``<color>``: The new color to assign. |color-input|
+
+**Examples**
+
+* ``[p]editrole color "My role" #ff0000``
+
+* ``[p]editrole color "My role" dark_blue``
+
+.. _admin-command-announce:
+
+^^^^^^^^
+announce
+^^^^^^^^
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announce <message>
+
+**Description**
+
+Announce your message to all of the servers the bot is in.
+
+The bot will announce the message in the guild's announcements channel
+if set, else she will try the system channel (where the new members are
+welcomed with the Discord announcer). If none of these channels are found,
+the bot will use the first one.
```

Comment:
The first what? Channel? Is it the first created, or the highest in the visual order?